.github/workflows/promote.yml

```diff
@@ -62,17 +62,33 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
 
+  # This job is renames the sfdx-x64.pkg that oclif v3 produces to sfdx.pkg which matches what oclif v2 produces.
+  # Doing so avoids breaking changes for customers relying on the old URLs to download the CLI.
+  # It can be removed once the developer site is linking to the new sfdx-x64.pkg file and the mac signing job is signing the new -x64.pkg file.
+  rename-mac-pkg:
+    runs-on: ubuntu-latest
+    env:
```

Comment:
how does this machine get aws?  or is it built into the ubuntu-latest?
https://github.com/salesforcecli/github-workflows/pull/10/files# doesn't do anything to make that happen.

the pack/upload mac job gets it via checkout, node, then yarn install (the CLI has it as a devDep)
https://github.com/salesforcecli/github-workflows/blob/60225d372a5ddca0fa936a6984e0d91057964bab/.github/workflows/packUploadMac.yml#L30

Reply:
Yeah, it's built into `ubuntu-latest`.

Reply:
https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#cli-tools